spring-cloud-alibaba-starters/spring-cloud-alibaba-governance/governance-core/spring-cloud-starter-alibaba-governance-labelrouting/label-routing-core/src/main/java/com/alibaba/cloud/router/ribbon/LabelRouteRule.java

```diff
@@ -154,88 +154,65 @@ private void serviceFilter(String targetServiceName, HashSet<String> versionSet,
 					.ofNullable(matchService.getRuleList());
 			Optional<String> version = Optional.ofNullable(matchService.getVersion());
 			Integer weight = matchService.getWeight();
+
 			if (weight == null) {
 				weight = 100;
 			}
-
-			if (doNotNullCheck(ruleList, version, weight, matchService,
-					targetServiceName)) {
+			if (!ruleList.isPresent() || ruleList.get().size() == 0) {
+				continue;
+			}
+			if (!version.isPresent()) {
+				LOGGER.warn("Target service ={} rule ={} lose version,please check it",
```

Comment:
Between '=' and '{}' need a blank. Please modify each place in your PR.

Reply:
done